wallet/public/index.html

```diff
@@ -60,19 +37,66 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Agoric Wallet</title>
+    <script type="module">
+      const MAX_ATTEMPTS = 10;
+      const WAIT_PER_ATTEMPT = 10;
+      let attempts = 0;
+      const startWhenKeplrAvailable = () => {
+        console.debug('looking for keplr');
+        attempts += 1;
+        if (window.keplr) {
+          // Allow the React dev environment to extend the console for debugging
+          // features.
+          const consoleTaming =
+            '%NODE_ENV%' === 'production' ? 'safe' : 'unsafe';
+
+          const { pow: mathPow } = Math;
+          Math.pow = (base, exp) =>
+            typeof base === 'bigint' && typeof exp === 'bigint'
+              ? base ** exp
+              : mathPow(base, exp);
+
+          lockdown({
+            errorTaming: 'unsafe',
+            overrideTaming: 'severe',
+            consoleTaming: consoleTaming,
+          });
+          console.log('lockdown done.');
+
+          window.addEventListener('unhandledrejection', ev => {
+            ev.stopImmediatePropagation();
+          });
+
+          console.log('Importing app bundle');
+          import('<%= htmlWebpackPlugin.files.js[0] %>');
```

Comment:
Do you think this might break if we ever do code-splitting? I'm not sure how webpack really works here, just flagging in case.

Reply:
yes it would. I'll make a note